### ISSUE TITLE
CodeSniffer bookmarklet throws an error on itself?

### ISSUE THREAD

**Issue by mattpolicastro** [squizlabs/HTML_CodeSniffer#130]
Was running the bookmarklet on my site (set to WCAG2AAA) and got an error I hadn't seen before: 

"Link element is missing a non-empty href attribute pointing to the resource being linked."

The bookmarklet says it's outside the document body so it can't point to it, but the code snippet preview shows:

````
...,s) })(document); </script><link rel="stylesheet" media="all"> <script src="//squizlabs.githu...
````

Am I crazy, or does is the bookmarklet throwing an error on itself?

**Comment by lwright-sq** [comment]
Hi Matt,

Thanks for the report. In short, no it's not.

The bookmarklet only inserts its script and stylesheet link tag into the end of the head tag's contents - which is what you're seeing in the in-context snippet preview. (If you look at the source view after loading HTMLCS, this should be the last two tags in your head section.)

That empty link tag would otherwise be the last thing in your head tag, and is what the error is referring to (ie. your stylesheet link tag there has no stylesheet URL).

**Comment by mattpolicastro** [comment]
Apologies—I ended up pinning this on a misbehaving TypeKit script, and forgot to close. Thanks for touching base.
